tensorflow/python/keras/layers/wrappers.py

```diff
@@ -22,6 +22,7 @@
 import copy
 
 from tensorflow.python.framework import tensor_shape
+from tensorflow.python.framework.ops import convert_to_tensor
```

Comment:
Please just import the framework.ops, and use the function like ops.convert_to_tensor. This will avoid the confusion for whether this locally defined function.